op-challenger/scripts/alphabet/init_game.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -euo pipefail
+
+DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+cd "$DIR"
+
+cd ../../
+
+make
+
+cd ..
+
+echo "----------------------------------------------------------------"
+echo " - Cleaning up the devnet"
+echo "----------------------------------------------------------------"
+
+make devnet-clean
+
+echo "----------------------------------------------------------------"
+echo " - Starting a new devnet"
+echo "----------------------------------------------------------------"
+
+make cannon-prestate
+make devnet-up
+
+DEVNET_SPONSOR="ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+DISPUTE_GAME_PROXY=$(jq .DisputeGameFactoryProxy .devnet/addresses.json)
+DISPUTE_GAME_PROXY=$(echo $DISPUTE_GAME_PROXY | tr -d '"')
```

Comment:
```suggestion
DISPUTE_GAME_PROXY=$(jq -r .DisputeGameFactoryProxy .devnet/addresses.json)
```

should work here too